language/testing-infra/e2e-tests/src/executor.rs

```diff
@@ -74,7 +73,7 @@ impl FakeExecutor {
 
     pub fn allowlist_genesis() -> Self {
         Self::custom_genesis(
-            stdlib_modules(StdLibOptions::Compiled).to_vec(),
+            stdlib_modules(StdLibOptions::Compiled).0.unwrap(),
```

Comment:
I just don't like the `.0`s in the PR. Other than that seems pretty reasonable to me. Should we rename this to `stdlib_modules_bytes` as what you did later? Or just provide this api in `compiled_stdlib`? I understand we need the tuple to make sure that there's correspondance between the two but the `.0` here is making me feel a bit uneasy.

Reply:
+1, should add stdlib_modules_bytes for .0, and something like `stdlib_compiled_modules`  for .1

Reply:
I'm purposefully not splitting the APIs to prevent misuse. I will improve upon the tuple here though